wordcount.py

```diff
@@ -39,16 +39,34 @@
 
 import sys
 
-# +++your code here+++
-# Define print_words(filename) and print_top(filename) functions.
-# You could write a helper utility function that reads a file
-# and builds and returns a word/count dict for it.
-# Then print_words() and print_top() can just call the utility function.
+def print_words(filename):
+    import operator
+    d = word_dict(filename)
+    sorted_d = sorted(d.items(), key = operator.itemgetter(0))
+    for k in sorted_d:
+        print k[0], k[1]
+
+def print_top(filename):
+    import operator
+    d = word_dict(filename)
+    sorted_d = sorted(d.items(), key=operator.itemgetter(1))
```

Comment:
You can sort the dictionary reversed, using the `reverse=True` additional parameter.